core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -431,12 +431,7 @@ public LockedInodePath lockInodePath(AlluxioURI uri, LockPattern lockPattern, bo
       throws InvalidPathException {
     LockedInodePath inodePath =
         new LockedInodePath(uri, mInodeStore, mInodeLockManager, getRoot(), lockPattern, tryLock);
-    try {
-      inodePath.traverse();
-    } catch (Throwable t) {
-      inodePath.close();
-      throw t;
-    }
+    inodePath.traverseOrClose();
```

Comment:
This code is functionally identical to the prior code, but we use `traverseOrClose()` to DRY-ify it.

Reply:
Nevermind, `traverseOrClose()` is `private` so this change is not trivial. Reverted.